### ISSUE TITLE
[Feature] Show Color Contrast Ratio

### ISSUE THREAD

**Issue by OLLI-S** [webhintio/hint#2746]
# 🚀 Feature request

## Description

 What do you want to be added?
It would be very helpful to see the Color Contrast Ratio of the element (like 3.4:1).

 Are you willing to work on this?
I am not a developer, sorry...

## What scenarios will this solve?
When there are elements with insufficient color contrast, I have to measure the color contrast manually and tell the developers "the color contrast is just 3.4:1, it must be at least 4.5:1).

 Why do you want/need this?
It saves me and my colleagues a lot of work!

**Comment by antross** [comment]
Thanks @OLLI-S for the suggestion! I agree this would be helpful.

I still need to dig into the code, but I think the contrast ratio is available as part of additional information from `axe-core` (which powers webhint's accessibility tests). It's not exposed as part of the current message, but surfacing this additional data from `axe-core` would probably be helpful for both this and other scenarios as well.

**Comment by antross** [comment]
For now, I'm going to try just appending this to the current report message. Then we can adjust if needed if that looks too busy or overwhelming.
